From 37a681596b451c4f6506312cb27efb4ca14e876d Mon Sep 17 00:00:00 2001
From: Tobias Fella <fella@posteo.de>
Date: Wed, 17 Feb 2021 23:23:13 +0100
Subject: [PATCH] Fix include

---
 src/messageeventmodel.cpp | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/src/messageeventmodel.cpp b/src/messageeventmodel.cpp
index 636c95c..36c4022 100644
--- a/src/messageeventmodel.cpp
+++ b/src/messageeventmodel.cpp
@@ -11,10 +11,11 @@
 #include <events/redactionevent.h>
 #include <events/roomavatarevent.h>
 #include <events/roommemberevent.h>
-#include <events/stickerevent.h>
 #include <events/simplestateevents.h>
 #include <user.h>
 
+#include "stickerevent.h"
+
 #include <QDebug>
 #include <QQmlEngine> // for qmlRegisterType()
 #include <QTimeZone>

From b67a35bfe34f95c35b7805147941b0c0e8635697 Mon Sep 17 00:00:00 2001
From: Tobias Fella <fella@posteo.de>
Date: Wed, 17 Feb 2021 23:26:52 +0100
Subject: [PATCH] Fix another include

---
 src/neochatroom.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/neochatroom.cpp b/src/neochatroom.cpp
index d468581..8593823 100644
--- a/src/neochatroom.cpp
+++ b/src/neochatroom.cpp
@@ -27,7 +27,7 @@
 #include "events/roomcanonicalaliasevent.h"
 #include "events/roommessageevent.h"
 #include "events/roompowerlevelsevent.h"
-#include "events/stickerevent.h"
+#include "stickerevent.h"
 #include "events/typingevent.h"
 #include "jobs/downloadfilejob.h"
 #include "neochatconfig.h"

